api/content_node_monitor.go

```diff
@@ -114,7 +114,7 @@ func (m *ContentNodeMonitor) getRegisteredNodes(ctx context.Context) ([]config.N
 	var nodes []config.Node
 	var endpointStrings []string
 	for _, endpoint := range endpoints.Msg.Endpoints {
-		if endpoint.ServiceType == "content-node" {
+		if endpoint.ServiceType == "content-node" || endpoint.ServiceType == "validator" {
 			nodes = append(nodes, config.Node{
 				Endpoint:            endpoint.Endpoint,
 				DelegateOwnerWallet: endpoint.DelegateWallet,
```

Comment:
cc @alecsavvy 
On line 122, we have IsStorageDisabled hard-coded to `false` because the core rpc doesn't return the value. Can there be validators returned here that don't support storage? If so, we need to get that plumbed through.